src/components/ShoppingCart/ShoppingCart.component.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+
+import ShoppingCartTable from './ShoppingCartTable/ShoppingCartTable.styles';
+import { useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+
+import { checkoutActionCreators } from 'state/index';
+import { bindActionCreators } from 'redux';
+import { useDispatch } from 'react-redux';
+
+const ShoppingCartComponent = ({className}) => {
+    const { items } = useSelector((state) => state.shoppingCart);
+    const dispatch = useDispatch()
+    const { setCheckoutSummary } = bindActionCreators(checkoutActionCreators, dispatch);
+    const history = useHistory();
+
+    const calculateTotalPrice = (items) => {
+        const totalPrice = items.reduce( (previous, current) => previous + (current.price*current.quantity),0);
+        return totalPrice;
+    }
+
+    const proceedToCheckout = (items) => {
```

Comment:
and it is the same for this one
```suggestion
    const proceedToCheckout = () => {
```